run-platform.sh

```diff
@@ -163,15 +163,19 @@ setup_env() {
           sed -i "s/ENCRYPTION_KEY.*/ENCRYPTION_KEY=\"$ENCRYPTION_KEY\"/" $env_path
         fi
       fi
-      # Add default auth credentials for backend.
+      # Add default auth and system admin credentials for backend.
       if [ "$service" == "backend" ]; then
-        echo -e "$blue_text""Adding default auth credentials to $service""$default_text"
+        echo -e "$blue_text""Adding default auth and system admin credentials to $service""$default_text"
         if [[ "$OSTYPE" == "darwin"* ]]; then
           sed -i '' "s/DEFAULT_AUTH_USERNAME.*/DEFAULT_AUTH_USERNAME=\"$DEFAULT_AUTH_KEY\"/" $env_path
           sed -i '' "s/DEFAULT_AUTH_PASSWORD.*/DEFAULT_AUTH_PASSWORD=\"$DEFAULT_AUTH_KEY\"/" $env_path
+          sed -i '' "s/SYSTEM_ADMIN_USERNAME.*/SYSTEM_ADMIN_USERNAME=\"$DEFAULT_AUTH_KEY\"/" $env_path
+          sed -i '' "s/SYSTEM_ADMIN_PASSWORD.*/SYSTEM_ADMIN_PASSWORD=\"$DEFAULT_AUTH_KEY\"/" $env_path
```

Comment:
@muhammad-ali-e what will this lines do? I don't think we need to configure this in the scripts since we are not yet planning to enable the admin part.

Reply:
This values are mandatory for initial migrations.